main.py

```diff
@@ -2,6 +2,29 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    names_of_genres = ["Western", "Action", "Dramma"]
+    for name in names_of_genres:
+        Genre.objects.create(name=name,)
+
+    names_of_actors = [
+        ("George", "Klooney"), ("Kianu", "Reaves"), ("Scarlett", "Keegan"),
+        ("Will", "Smith"), ("Jaden", "Smith"), ("Scarlett", "Johansson")
+    ]
+    for first_name, last_name in names_of_actors:
+        Actor.objects.create(first_name=first_name, last_name=last_name)
```

Comment:
No need to fix it, but for future - .bulk_create() can be used to create multiple instances in 1 db query